vagrant/package-base-box.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# This script automates the process of creating and packaging 
+# a new vagrant base_box. For use locally (not aws).
+
+# Name of the new base box
+base_box="kafkatest-worker"
+
+# vagrant VM name
+worker_name="worker1"
+
+base_dir=`dirname $0`/..
+cd $base_dir
+
+echo "Destroying vagrant machines..."
+vagrant destroy -f
+
+echo "Removing $base_box from vagrant..."
+vagrant box remove $base_box
+
+echo "Bringing up a single vagrant machine from scratch..."
+backup_vagrantfile=backup_Vagrantfile.local
+local_vagrantfile=Vagrantfile.local
+if [ -e $local_vagrantfile ]; then
+    mv $local_vagrantfile $backup_vagrantfile
+fi
+echo "num_workers = 1" > $local_vagrantfile
+echo "num_brokers = 0" >> $local_vagrantfile
+echo "num_zookeepers = 0" >> $local_vagrantfile
+vagrant up
+up_success=$?
+if [ $up_sucess != 0 ]; then
```

Comment:
Got this output when I ran this:
```
vagrant/package-base-box.sh: line 32: [: !=: unary operator expected
Packaging worker1...
```
I think you just have a typo: `up_sucess` -> `up_success`